### DISCUSSION TITLE
WJLP-DT8 MeTV+

### DISCUSSION THREAD

**Discussion by Gitbone94** [iptv-org/community#866] [Q&A]
### Channel ID

WJLPDT8.us

### Notes

WJLP DT1-7 are already in the lineup as shown in the epg channel search:
https://iptv-org.github.io/epg/guides/en/tvpassport.com.xml
It looks like DT-8 is new, so please add if possible.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19839
